content/200-concepts/100-components/01-prisma-schema/06-relations/420-referential-integrity.mdx

```diff
@@ -72,13 +72,13 @@ and was previously named <inlinecode>referentialIntegrity</inlinecode>.
 The available options are:
 
 - `foreignKeys`: this handles referential integrity in the database using foreign keys. This is the default option when using all SQL database connectors.
-- `prisma`: this emulates referential integrity in the Prisma client. This is the default (and only available) option when using the MongoDB connector. You should also [enable this option](/guides/database/using-prisma-with-planetscale#how-to-enable-emulation-of-referential-integrity) when using the MySQL connector with a PlanetScale database.
+- `prisma`: this emulates relations in the Prisma client. This is the default (and only available) option when using the MongoDB connector. You should also [enable this option](/guides/database/using-prisma-with-planetscale#how-to-enable-emulation-of-referential-integrity) when using the MySQL connector with a PlanetScale database.
 
 <Admonition type="warning">
-  When using Prisma Migrate, setting the value to `prisma` will remove all
-  previously created foreign keys in the next migration. Setting the value to
-  `foreignKeys` will create foreign keys for all relations in the next
-  migration.
+
+When using Prisma Migrate, setting the value to <inlinecode>prisma</inlinecode> will remove all previously created foreign keys in the next migration. Setting the value to <inlinecode>foreignKeys</inlinecode> will create foreign keys for all relations
```

Comment:
foreignKeys or default, as in remove the relationMode property

Reply:
Updated to clarify this.